.travis.yml

```diff
@@ -8,8 +17,14 @@ before_install:
 cache:
   yarn: true
 
+install:
+  - export DISPLAY=':99.0'
+  - Xvfb :99 -screen 0 1920x1080x24 > /dev/null 2>&1 &
```

Comment:
The screen has to be bigger than the one specified in the documentation or a 800 x 600 window will be resized to fill into the screen :S